yorkie/src/main/kotlin/dev/yorkie/document/crdt/RHTPQMap.kt

```diff
@@ -0,0 +1,161 @@
+package dev.yorkie.document.crdt
+
+import android.util.Log
```

Comment:
I wonder if there is a way not to add the Android dependency directly here. It looks like we could create a Logger interface. 🤔

```suggestion
import android.util.Log
```

Reply:
Changed it to use custom logger interface : [5ae255e](https://github.com/yorkie-team/yorkie-android-sdk/pull/10/commits/5ae255e1faff0e9b5bd764b2d285dba05c21bcf0)
When developer want to use android log, then set `andorid.util.Log` to `YorkieLogger`